docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -63,3 +63,48 @@ For example:
 
 The same applies to `slotProps` and `componentsProps`.
 :::
+
+### Adapters
+
```

Comment:
```suggestion
### Adapters

The adapter does the glue between the date library you chose and the pickers component.
Standard usage of the library should not include those adapters.
Those breaking changes are listed for users that have advanced usage such as creating a library based on the pickers.

```

Maybe adding a text to say that most of the user don't have to care about this section

Reply:
Oh sorry, I did a brief introduction in #10971, I'll let you review there if you want to improve the content :+1:

Reply:
No problem, your description is nice in this other PR :+1: